asset/s3_exporter/processor.js

```diff
@@ -99,12 +106,25 @@ class S3Batcher extends BatchProcessor {
             throw new Error(`Unsupported output format "${this.opConfig.format}"`);
         }
 
+        // This will prevent empty objects from being added to the S3 store, which can cause
+        // problems with the S3 reader
+        if (outStr.length === 0) {
+            return Promise.resolve();
```

Comment:
The batch processor needs to return an array. Side note, when a function is marked with the `async` keyword, a Promise always returned so using `return Promise.resolve(...)` is redundant.

```suggestion
            return [];
```

Reply:
fixed